### ISSUE TITLE
v1.9.0 creates invalid json by including undefined values

### ISSUE THREAD

**Issue by src-code** [yahoo/serialize-javascript#57]
There seems to be a regression in v1.9.0 where objects with undefined values are being included in the json output, which is invalid.  For example:

```js
serialize({
  "foobar": undefined
});
```

Output:

```json
{"foobar":undefined}
```

This is not valid json, and will fail when run through `JSON.parse()`

In v1.8.0 this worked correctly with the following output:

```json
{}
```

**Comment by src-code** [comment]
I guess PR #54 was trying to typecast `undefined` to a string, but it doesn't seem to be working in the above example?

**Comment by src-code** [comment]
Also, I'm not sure it's a good idea to cast undefined into a string, this could lead to unexpected results for some implementations.  Should be something you can opt-in to if anything.

**Comment by src-code** [comment]
Closing this as I guess it's intentional that this lib creates a superset of json

**Comment by p-j** [comment]
Could you reopen as this looks like a breaking change and should have prompted a major version bump.

**Comment by okuryu** [comment]
I just released v1.9.1 (includes reverting #54) and v2.0.0 for this issue. Thanks!
- https://github.com/yahoo/serialize-javascript/releases/tag/v1.9.1
- https://github.com/yahoo/serialize-javascript/releases/tag/v2.0.0

**Comment by p-j** [comment]
Thanks for the quick fix @okuryu !
